### PR TITLE
Split serving into two separate work groups: observability and networking

### PR DESCRIPTION
**Pull request by mdemirhan**
Split serving into two separate work groups: observability and networking

### CODE REVIEW

File: community/WORKING-GROUPS.md [hunk 1]
```diff
@@ -78,41 +79,60 @@ Slack Channel              | [#eventing](https://knative.slack.com/messages/C9JP
...
 Artifact                   | Link
 -------------------------- | ----
 Forum                      | [knative-dev@](https://groups.google.com/forum/#!forum/knative-dev)
-Community Meeting VC       | [scaling-wg](https://hangouts.google.com/hangouts/_/google.com/scaling-wg)
-Community Meeting Calendar | [Calendar](https://calendar.google.com/calendar?cid=Z29vZ2xlLmNvbV9sNHY5NnJocWJoMGk1dWsyMnU1MGhudDY2c0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t)
-Meeting Notes              | [Notes](https://docs.google.com/document/d/1FoLJqbDJM8_tw7CON-CJZsO2mlF8Ia1cWzCjWX8HDAI/edit#heading=h.c0ufqy5rucfa)
-Document Folder            | [Folder](https://drive.google.com/corp/drive/folders/1qpGIPXVGoMm6IXb74gPrrHkudV_bjIZ9)
-Slack Channel              | [#autoscaling](https://knative.slack.com)
+Community Meeting VC       | Coming soon
+Community Meeting Calendar | Coming soon
```
Comment (mattmoor) [inline community/WORKING-GROUPS.md]: We should make sure Istio has a presence here.

File: community/WORKING-GROUPS.md [hunk 2]
```diff
@@ -78,41 +79,58 @@ Slack Channel              | [#eventing](https://knative.slack.com/messages/C9JP
...
+Meeting Notes              | [Notes](https://drive.google.com/open?id=1EE1t5mTfnTir2lEasdTMRNtuPEYuPqQCZbU3NC9mHOI)
+Document Folder            | [Folder](https://drive.google.com/corp/drive/folders/1oVDYbcEDdQ9EpUmkK6gE4C7aZ8u6ujsN)
```
[the remark this answers is not in the document]
Comment (mattmoor) [reply community/WORKING-GROUPS.md]: Generally this split LGTM, but I would like to see us beef up the roadmap (duration and detail).  I left some thoughts on the doc.

**Comment by evankanderson** [comment]
@vaikas-google just went on vacation after GCP Next; I'm going to time out his vote and approve in the next 24 hours.

**Comment by evankanderson** [comment]
(This now has merge conflicts)

**Pull request closed without merging**